scripts/generate_embeddings_openstack.py

```diff
@@ -50,106 +51,24 @@ def url_function(self, file_path: str):
         )
 
 
-def get_argument_parser() -> argparse.ArgumentParser:
-    parser = argparse.ArgumentParser(description="Generate vector DB embeddings!")
-
-    # TODO(lpiwowar): We should be able to support building of a single vector database
-    #                 for both upstream and downstream documentation
-    group = parser.add_mutually_exclusive_group(required=True)
-    group.add_argument(
+if __name__ == "__main__":
+    parser = utils.get_common_arg_parser()
+    parser.add_argument(
         "-rf",
         "--rhoso-folder",
         type=Path,
         required=False,
         help="Directory containing the plain text RHOSO documentation",
     )
-    group.add_argument(
```

Comment:
:+1: I verified this chunk against https://github.com/road-core/rag-content/blob/main/src/lightspeed_rag_content/utils.py#L19